plugins/Print.jsx

```diff
@@ -545,7 +544,7 @@ class Print extends React.Component {
     print = (ev) => {
         if (this.props.inlinePrintOutput) {
             this.setState({printOutputVisible: true, outputLoaded: false});
-        } else {
+        }
```

Comment:
Can you fix the indentation of the following lines?

Reply:
The subsequent lines should be de-indented as the indentation block was removed.

Reply:
MB sorry, should be fixed